src/big/big_decimal.cr

```diff
@@ -257,6 +257,20 @@ struct BigDecimal
     end
   end
 
+  # Factors out any extra powers of ten in the internal representation.
+  # For instance, value=100 scale=2 => value=1 scale=0
+  def factor_powers_of_ten
```

Comment:
Shouldn't this be `private` and called only from inside `BigDecimal`?

Reply:
Ow, that was fast reactions :) I guess so. That makes it impossible to unit test it, though, doesn't it?